omnibus/config/chefdk_overrides.rb

```diff
@@ -0,0 +1,23 @@
+override :berkshelf, :version => "v4.3.1"
```

Comment:
these are also intended to grow into non-gem overrides as well?

either way from the omnibus perspective they'd be perfectly interchangable and not add anything so i'd be :-1:, it'd be better to just name the file chefdk_gem_overrides.rb